sensing/autoware_pointcloud_preprocessor/schema/crop_box_filter_node.schema.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Crop Box Filter Node",
+  "type": "object",
+  "definitions": {
+    "crop_box_filter": {
+      "type": "object",
+      "properties": {
+        "min_x": {
+          "type": "number",
+          "description": "x-coordinate minimum value for crop range",
+          "default": "-1.0"
+        },
+        "min_y": {
+          "type": "number",
+          "description": "y-coordinate minimum value for crop range",
+          "default": "-1.0"
+        },
+        "min_z": {
+          "type": "number",
+          "description": "z-coordinate minimum value for crop range",
+          "default": "-1.0"
+        },
+        "max_x": {
+          "type": "number",
+          "description": "x-coordinate maximum value for crop range",
+          "default": "1.0"
+        },
+        "max_y": {
+          "type": "number",
+          "description": "y-coordinate maximum value for crop range",
+          "default": "1.0"
+        },
+        "max_z": {
+          "type": "number",
+          "description": "z-coordinate maximum value for crop range",
+          "default": "1.0"
+        },
+        "negative": {
+          "type": "boolean",
+          "description": "if negative then choose the point outside the box",
```

Comment:
write this one better please

Reply:
Thanks! fixed in https://github.com/autowarefoundation/autoware.universe/pull/8466/commits/772c90251b465caeab41535c50fc12f39ab10a32